params/three_limit_circles.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "three_limit_circles",
+    "name": "Three Limit Circles",
+    "algorithm": "chaos",
+    "plotter": {
+        "radius": 4
+    },
+    "cluster": {
```

Comment:
keep trying different clusters here. this fractal is neat but see what's the best way to visualize it

Reply:
I found I like it the way it currently is.